### DISCUSSION TITLE
Some QoL gameplay skin element ideas

### DISCUSSION THREAD

**Discussion by SIUUUUUUUUUU17** [ppy/osu#30283] [Ideas]
**1. Hide any element and/or show completely different text if a {Value} is false or true**

For example, I want to hide the TextAttribute element if the map status is Ranked, and show a text that says "This beatmap will not award PP" if it's not, to indicate whether I'm playing a Loved or Graveyard map in my recordings

**2. Mod list, either staggered or not, vertically or horizontally**

**3. Customizing the animation for the Argon counter**

Say, I want it to pop in size very quickly, flash whether a combo milestone is hit (Kind of like kiai), and also immediately count up combo and not gradually like present. There should be an option to hide the counter if less than a minimum combo number, too

**4. More TextAttribute attributes**

Having a `Combo` attribute helps make No. 3 a little more plausible, also a `PlayRankedStatus` attribute and a `ScoreMultiplier` attribute to know how severe your score is affected by certain mods

**Comment by shinmorisawa** [comment]
Would be kinda nice ya know
